modules/core/src/core/animation-loop.js

```diff
@@ -1,13 +1,37 @@
 /* global OffscreenCanvas */
+/* global performance, process */
+
 import {createGLContext, resizeGLContext, resetParameters} from '../webgl/context';
 import {getPageLoadPromise} from '../webgl/context';
 import {isWebGL, requestAnimationFrame, cancelAnimationFrame} from '../webgl/utils';
 import {log} from '../utils';
 import assert from '../utils/assert';
+import {Stats} from 'probe.gl';
+import {Query} from '../webgl';
 
 // TODO - remove dependency on webgl classes
 import {Framebuffer} from '../webgl';
 
+const USE_PERFORMANCE = typeof performance !== 'undefined';
+const USE_HRTIME = typeof process !== 'undefined';
+let statIdCounter = 0;
+
+// TODO - Remove when available from probe.gl
```

Comment:
Can we just publish a new version and sort this TODO before we land?

Reply:
I believe @Pessimistress did publish it. I'll check if it's available.